.github/PULL_REQUEST.md

```diff
@@ -3,3 +3,6 @@
 
 ## Why
 * Provide the justifications for the changes.
+
+## Integration Tests
+* Before requesting review, please run integration tests on your branch by making a new commit and including `[build-integration-tests]` in the commit message. Our TravisCI configuration will recognize this string and trigger integration tests. It's pretty cool. Afterwards, please indicate the relevant commit below, using either the commit message or truncated commit hash. (Note that triggering integration tests manually isn't necessary if you're merging from `stable` into `master`.)
```

Comment:
It seems a bit wordy compared to the other text in the template. 

## Integration Tests
* Run integration testing by including `[build-integration-tests] in the commit message.
* Integration tested commit hash: `(         )`

Reply:
Implemented!